lib/managers/users.js

```diff
@@ -78,6 +79,37 @@ Users.prototype.update = function(id, options, callback) {
 };
 
 /**
+ * Invites an existing user to join an Enterprise. The existing user can not be
+ * part of another Enterprise and must already have a Box account. Once invited,
+ * the user will receive an email and prompt to accept the invitation within
+ * the Box web application. This method requires the "Manage An Enterprise"
+ * scope for the enterprise, which can be enabled within your developer console.
+ *
+ * API Endpoint: '/invites'
+ * Method: POST
+ *
+ * @param {string} enterpriseID - The enterprise the user will be invited to
+ * @param {string} login - The login of the user that will receive the invitation
+ * @param {Function} callback - Passed the invitation information if it was acquired successfully, error otherwise
+ * @returns {void}
+ */
+Users.prototype.invite = function(enterpriseID, login, callback) {
```

Comment:
Sure does the same apply for Create User for example?[ There is a comment in the code](https://github.com/box/box-node-sdk/blob/master/lib/managers/users.js#L135-L138) which seems to be related to this.